system-setting/backend/src/main/java/io/metersphere/service/SystemProjectService.java

```diff
@@ -123,7 +123,10 @@ public Project addProject(AddProjectRequest project) {
         // 设置默认的通知
         baseProjectMapper.setDefaultMessageTask(project.getId());
 
-        quotaService.projectUseDefaultQuota(pjId);
+        if (quotaService!=null) {
```

Comment:
format

Reply:
Thanks for your suggestion, I'll take care of this